### PR TITLE
feat: consolidated catalog filters

### PR DESCRIPTION
**Pull request by kamilczaja**
_What issues does this PR close?_

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
   - Already registered connectors will be updated automatically, this process can take up to 24 hours
 - Added a message when the CaaS request feature is not available
 - Catalog: Removed dataspace filter when only one dataspace is known
+- Catalog: Merged Organization ID and name filter into one
```
Comment (richardtreier) [inline CHANGELOG.md:25]: `- Catalog: Connector Filter now also displays names and organization names`

**Review by richardtreier — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kamilczaja) [reply CHANGELOG.md:25]: That doesn't really describe the change though. The names were there before, they were just separate filters

File: authority-portal-backend/authority-portal-quarkus/src/test/kotlin/de/sovity/authorityportal/web/tests/services/catalog/CatalogApiTest.kt [hunk 2]
```diff
@@ -335,25 +243,42 @@ class CatalogApiTest {
...
-            dataOffer(0, 0, 3) {
-                it.assetTitle = "Data Offer 3"
-                it.uiAssetJson = JSONB.valueOf(objectMapper.writeValueAsString(uiAsset3))
-            }
-            dataOffer(0, 0, 4) {
-                it.assetTitle = "Data Offer 4"
-                it.uiAssetJson = JSONB.valueOf(objectMapper.writeValueAsString(uiAsset4))
-            }
+            dataOffer(0, 0, 0, assetApplier = {
+                it.assetId = dummyDevAssetId(0)
```
[the remark this answers is not in the document]
Comment (kamilczaja) [reply authority-portal-backend/authority-portal-quarkus/src/test/kotlin/de/sovity/authorityportal/web/tests/services/catalog/CatalogApiTest.kt:247]: Yup

File: authority-portal-frontend/src/app/pages/empty-pages/unauthenticated-page/unauthenticated-page/unauthenticated-page.component.ts [hunk 3]
```diff
@@ -22,7 +22,7 @@ export class UnauthenticatedPageComponent {
 
   get loginUrl(): string {
     const url = new URL(this.appConfig.loginUrl);
-    url.searchParams.set('redirect_uri', location.href);
+    url.searchParams.set('rd', location.href);
```
[the remark this answers is not in the document]
Comment (kamilczaja) [reply authority-portal-frontend/src/app/pages/empty-pages/unauthenticated-page/unauthenticated-page/unauthenticated-page.component.ts:25]: Didn't work anyway, might as well revert it. Will be fixed in another PR

**Review by richardtreier — APPROVED** [review]
LGTM

**Pull request merged**